apps/src/templates/rubrics/introjs.scss

```diff
@@ -445,4 +445,3 @@ tr.introjs-showElement > th {
   z-index: 1;
   opacity: 0;
 }
-/*# sourceMappingURL=introjs.css.map */
```

Comment:
Happened to you too :( 

I just merged the fix, so merging in staging should fix.

Reply:
d'oh! Thanks for the catch